File: Dataset/Leetcode/valid/125/448.cpp
 class Solution {
public:
int CheckChar(char x){
    if(x>=65&&x<=90) return 1;
    if(x>=97&&x<=122) return 2;
    if(x>=48&&x<=57) return 3;
    return 0;
}
string modify(string x){
	string result="";
	for(int i=0;i<x.length();i++){
		if(!CheckChar(x[i])) continue;
		if(CheckChar(x[i])==1) x[i]+=32;
		result+=x[i];
	}
	return result;
} 

bool XXX(string s) {
	for(int i=0;i<s.length()/2;i++) if(s[i]!=s[s.length()-1-i]) return false;
	return true;
}
};

File: Dataset/Leetcode/test/46/448.py
 class Solution:
    def XXX(self, nums: List[int]) -> List[List[int]]:
        res = []
        track = []
        def backtrack(nums, track):
            if len(nums) == 0:
                res.append(track.copy())
            for i in range(len(nums)):
                track.append(nums[i])
                temp_nums = nums.copy()
                nums.pop(i)
                backtrack(nums, track)
                track.pop()
                nums = temp_nums.copy()
        backtrack(nums, track)
        return res


